lib/vulcan.js

```diff
@@ -42,10 +42,26 @@ function isExcludedImport(importMeta) {
   });
 }
 
-function replaceWith(parent, node, replacements) {
-  var idx = parent.childNodes.indexOf(node);
-  parent.childNodes = parent.childNodes.slice(0, idx).
-    concat(replacements, parent.childNodes.slice(idx + 1));
+function isBlankTextNode(node) {
+  return node && dom5.isTextNode(node) && !/\S/.test(dom5.getTextContent(node));
```

Comment:
No, that would be if any character was a whitespace.
`!/\S/` is if any character *isn't* whitespace.